apps/api/package.json

```diff
@@ -12,9 +12,9 @@
     "precommit": "lint-staged",
     "docker:build": "pnpm --silent --workspace-root pnpm-context -- apps/api/Dockerfile | docker build --build-arg PACKAGE_PATH=apps/api - -t novu-api",
     "start": "pnpm start:dev",
-    "start:dev": "cross-env TZ=UTC nest start --watch --debug",
+    "start:dev": "cross-env TZ=UTC nest start --watch",
     "start:test": "cross-env NODE_ENV=test PORT=1336 TZ=UTC nest start --watch",
-    "start:debug": "TZ=UTC nodemon --config nodemon-debug.json",
+    "start:debug": "TZ=UTC nodemon --config nodemon-debug.json --debug",
```

Comment:
```suggestion
    "start:debug": "cross-env TZ=UTC nest start --watch --debug",
```

Reply:
I wonder if should enable --debug by default for `start:dev` command 🤔 @LetItRock @p-fernandez what do you think?

Reply:
It does not hurt to have separate commands to be fair. Also I ignore if there could be conflicts with the debug flat by default with the set ups of any IDE user (not my problem though 😅 ).